### PR TITLE
Beatmap_Discussion: update from forum (Kudosu! has been implemented)

### LINKED ISSUE #868: Beatmap Discussion: update (again)
update the Beatmap Discussion article (again) <https://osu.ppy.sh/community/forums/topics/650961>

### PR DESCRIPTION
**Pull request by MegaApplePi**
resolves #868

### CODE REVIEW

File: wiki/Beatmap_Discussion/en.md [hunk 1]
```diff
@@ -126,14 +126,22 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 ### Functions
 
 - `Collapse all`/`Expand all` and the up arrow: These will show/hide posts.
-- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this.
+- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Guving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on suggestions and problems.**
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: Guving a thumbs up?

File: wiki/Beatmap_Discussion/en.md [hunk 2]
```diff
@@ -126,14 +126,22 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 ### Functions
 
 - `Collapse all`/`Expand all` and the up arrow: These will show/hide posts.
-- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this.
+- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on suggestions and problems.**
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: on sugestions *or* problems, maybe?

File: wiki/Beatmap_Discussion/en.md [hunk 3]
```diff
@@ -126,14 +126,22 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
 - Timeline: Shows the timestamp the issue addresses. Only available in the Timeline section.
 - CREATOR icon: Displays responses/submissions from the user who made the beatmapset.
 - permalink, edit, delete: Permalink grabs a direct link to the post. Edit allows the submitter to edit their submission. Delete allows the submitter to remove their submission.
 - Respond: Reply to the submission. Pressing Enter or clicking Reply afterwards will submit your response.
 - Mark as Resolved: The creator may select this when the issue has been addressed and either accepted or denied.
 - Reopen a Discussion: If the submission needs further discussion, the creator or mod submitter may reopen the submission by responding and unchecking Resolved.
 
+## Earning Kudosu!
+
+*Note: You are given ten votes every hour.*
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: either `*Note: You get ten votes every hour.*` or `*Note: You are given with ten votes every hour.*`

File: wiki/Beatmap_Discussion/en.md [hunk 4]
```diff
@@ -126,14 +126,22 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
+
+![](img/thumbs.jpg "Thumbs up for Kudosu!")
+
+Modders can now earn Kudosu! by having their mod be given the thumbs up. You can (and probably should) go through your old mods and give them a thumbs up to support the modders before this feature was implemented.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: by having a thumbs up on their mod

through your old mods -> through mods on your old beatmaps?

File: wiki/Beatmap_Discussion/en.md [hunk 5]
```diff
@@ -165,17 +173,22 @@ Your post will be submitted! You can restart at step 3 if you wish to continue m
 
 ### Is Kudosu! available in Beatmap Discussion?
 
-Kudosu! is not yet implemented in Beatmap Discussion. You will not recieve any for submissions you make in the Discussion section. However, it is unknown if current submissions will receive Kudsou! when it is finally implemented. It is also unknown whether or not Kudosu! will be returning either.
+Kudosu! has been implemented! You can give modders Kudosu! by using the thumbs Up icon on discussions that were marked as Suggestions or Problems.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: Kudosu! has been recently implemented in the Beatmap Discussion! You can give modders Kudosu! by using the thumbs up emote(?) on discussions that were marked as Suggestions or Problems.

File: wiki/Beatmap_Discussion/en.md [hunk 6]
```diff
@@ -165,17 +173,22 @@ Your post will be submitted! You can restart at step 3 if you wish to continue m
...
 
 ### How are beatmaps nominated?
 
 Beatmap Discussion follows the same procedure as its predecessor. 2 Beatmap Nominators are required to nominate the Meatmapset for qualification. Star Priority has been replaced with Hype and requires 12 Hype before it can be nominated.
 
+### Will I be able to mod with the older system after Beatmap Discussion releases?
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: after Beatmap Discussion is released

File: wiki/Beatmap_Discussion/en.md (hunk 3 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: going with `*Note: You get ten votes every hour.*`

File: wiki/Beatmap_Discussion/en.md (hunk 4 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: applied `by having a thumbs up on their mod`
...
`through your old mods` should be good enough, its talking about the mods before Kudosu! was implemented in beatmap discussion (not anything regards to old beatmaps)

Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: `through you old mods and give them a thumbs up` it means that you give thumb up to **your** own mods

Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: not other modders' mods

Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: alright, using `go through mods on your beatmaps` (your second suggestion, but removed `old` before `beatmaps`)

File: wiki/Beatmap_Discussion/en.md [hunk 7]
```diff
@@ -126,14 +126,22 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
 - Mark as Resolved: The creator may select this when the issue has been addressed and either accepted or denied.
 - Reopen a Discussion: If the submission needs further discussion, the creator or mod submitter may reopen the submission by responding and unchecking Resolved.
 
+## Earning Kudosu!
+
+*Note: You get ten votes every hour.*
+
+![](img/thumbs.jpg "Thumbs up for Kudosu!")
+
+Modders can now earn Kudosu! by having a thumbs up on their mod. You can (and probably should) go through your old mods and give them a thumbs up to support the modders before this feature was implemented.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: by having a thumb up on their mods?

Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: I forgot to press `Ctrl` + `S` (like for the first time ever, lol)

Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: unlucky :trollface:

**Pull request merged**